CMakeLists.txt

```diff
@@ -208,6 +208,13 @@ if(DOXYGEN_FOUND)
     )
 endif(DOXYGEN_FOUND)
 
+########################################################################
+# Detect /lib versus /lib64
+########################################################################
+if (CMAKE_INSTALL_LIBDIR MATCHES lib64)
```

Comment:
CMake is confusing as to whether the left side can be `CMAKE_INSTALL_LIBDIR` or `${CMAKE_INSTALL_LIBDIR}`. I'd prefer the latter, since that makes it clear that we mean the contents of the variable and not the variable name itself.

Reply:
Let me look. This patch is 4 years old and worked for a powerpc64 OE build and seems to work on my F3x machine. IS this what you are referring to? 
'''
The if command was written very early in CMake’s history, predating the ${} variable evaluation syntax, and for convenience evaluates variables named by its arguments as shown in the above signatures.
'''

Reply:
Updated.